app/dao/notifications_dao.py

```diff
@@ -267,6 +267,31 @@ def get_notifications_for_job(
     return pagination
 
 
+def get_recent_notifications_for_job(
+    service_id, job_id, filter_dict=None, page=1, page_size=None
+):
+    print(f"FILTER_DICT AT DAO LEVEL {filter_dict}")
```

Comment:
`print` statement - should this be a more clear cut logging statement or just be removed entirely?

```suggestion
```